bgpd/bgp_vty.c

```diff
@@ -2820,64 +2819,6 @@ DEFUN(no_bgp_ebgp_requires_policy, no_bgp_ebgp_requires_policy_cmd,
 	return CMD_SUCCESS;
 }
 
-DEFPY(bgp_lu_uses_explicit_null, bgp_lu_uses_explicit_null_cmd,
```

Comment:
Yeah, we can't just remove this.  Why are we removing this command?

Reply:
This CLI cmd isn't released yet, so IMO we can change it, no?